app/components/Select/index.js

```diff
@@ -0,0 +1,25 @@
+/**
+*
+* Select
+*
+* This adds a z-index to the Select component so it appears above the half stars from the ReactStars component
+*
+*/
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import ReactSelect from 'react-select';
+import 'react-select/dist/react-select.css';
+
+const EditDisabledOverlay = styled(ReactSelect)`
+  z-index: 2;
+`
+
+function Select(props) {
```

Comment:
I don't understand how this works, how does it get un-disabled?

Reply:
Uh, this is a big naming mistake on my part. Sorry about that.